app/src/org/commcare/network/HttpRequestEndpointsMock.java

```diff
@@ -1,4 +1,4 @@
-package org.commcare.android.mocks;
+package org.commcare.network;
```

Comment:
Will you rename the class to exclude `Mock`? When I read 'mock' I assume I'm looking at non-prod code